internal/discoverer/zookeeper_test.go

```diff
@@ -96,14 +96,14 @@ func TestZookeeperDiscoverer(t *testing.T) {
 	msgChan := dis.Watch()
 
 	// create service
-	err = createZkService(conn, svcPath, `{"host":"127.0.0.1","port":1980,"weight":100}`)
+	err = createZkService(conn, svcPath, `[{"host":"127.0.0.1","port":1980,"weight":100}]`)
```

Comment:
Is it possible to provide multi nodes in the test data, like `[{"host":"127.0.0.1","port":1980,"weight":100}], ...`?

Reply:
Complex arrays are difficult to compare when using json, and using `JSONEq` directly will make the test unstable.

Reply:
What about sorting the output before the comparison?

Reply:
The serialization of JSON is deeply hidden, and it is difficult to intervene. And that part of the code is not used for testing, the sorting in it is not appropriate. Look: https://github.com/api7/apisix-seed/blob/main/internal/discoverer/zookeeper_test.go#L56